src/events/events.spec.ts

```diff
@@ -1,7 +1,5 @@
 import axios from 'axios';
-
 import MockAdapater from 'axios-mock-adapter';
```

Comment:
Noticed we have a typo here.

It seems to be more widespread, so will fix it in a follow-up PR.